tests/src/test/scala/com/nvidia/spark/rapids/GpuDeviceManagerSuite.scala

```diff
@@ -129,7 +129,13 @@ class GpuDeviceManagerSuite extends AnyFunSuite with BeforeAndAfter {
       .set("spark.memory.offHeap.enabled", "true")
       .set("spark.memory.offHeap.size", sparkOffHeapSizeStr)
     val rapidsConf = new RapidsConf(sparkConf)
-    val availableHostMem = toBytes("16g")
+    val deviceCount = Cuda.getDeviceCount
+    val availableHostMemPerDevice = toBytes("16g")
+    // We provide more mem if there are more devices so that the
+    // final limit calculates to the same value, so that we hit
+    // the same code path regardless of num devices and don't have
+    // to worry about eg. minMemoryLimit which is covered separately
+    val availableHostMem = availableHostMemPerDevice * deviceCount
```

Comment:
Are we not concerned that this might exceed the total available memory?

Reply:
Also the unit test is only testing the logic/math, not trying to actually allocate the host memory